2020/day15/solution.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import  annotations
+
+input = [15,5,1,4,7,0]
+
+def find_nth_number(starting : list[int], nth : int) -> int:
+    last_spoken = { value : [index] for index, value in enumerate(starting)}
+    round = len(starting)
+    last_number = starting[-1]
+    while round < nth:
```

Comment:
You can use a for loop instead since you increment `round` by one on each iteration - https://docs.python.org/3/library/functions.html#func-range will be helpful.

Reply:
Note that `round` is also a built-in: https://docs.python.org/3/library/functions.html#round
I used `turn` as an alternative.

Reply:
another good to know :)

and yea, you are right i should just use range()